### PR TITLE
Add wavelo bike sharing

### PR DESCRIPTION
**Pull request by fenuks**
Add [wavelo](https://en.wavelo.pl/) support. It is making use of [Social Bicycles](http://socialbicycles.com/) technology. This provider probably could be used for other cities employing this system.

### CODE REVIEW

File: pybikes/socialbicycles.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import json
+
+from pybikes.contrib import TSTCache
+from pybikes.utils import PyBikesScraper
+
+from .base import BikeShareStation, BikeShareSystem
+
+__all__ = ['SocialBicycles', 'SocialBicyclesStation']
```
Comment (eskerda) [inline pybikes/socialbicycles.py]: Not used anymore

File: pybikes/socialbicycles.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import json
+
+from pybikes.contrib import TSTCache
+from pybikes.utils import PyBikesScraper
+
+from .base import BikeShareStation, BikeShareSystem
```
Comment (eskerda) [inline pybikes/socialbicycles.py]: Relative imports can be done like on the previous lines, in this case
```
from pybikes import BikeShareSystem, BikeShareStation
```

File: pybikes/socialbicycles.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+__all__ = ['SocialBicycles', 'SocialBicyclesStation']
+
+BASE_URL = 'https://app.socialbicycles.com/api/networks/{uid}/hubs?page={page}&per_page='
+
+# Since most networks share the same hostname, there's no need to keep hitting
+# the endpoint on the same urls. This caches the feed for 60s
+cache = TSTCache(delta=60)
```
Comment (eskerda) [inline pybikes/socialbicycles.py]: This system does not need a Timestamped Test Cache (TSTCache). This cache system was just a hack / stupid idea to support systems where one url gives multiple feeds at once.

Comment (fenuks) [reply pybikes/socialbicycles.py]: Ah, my bad. I got it copied from other system implementation.

**Comment by fenuks** [comment]
I'm glad you find this useful! Fixed all issues you pointed out.

**Comment by eskerda** [comment]
LGTM, many thanks @fenuks!

**Comment by eskerda** [comment]
Humm, just found out about this: https://wavelo.pl/opendata/gbfs.json

Still, it's good to support an alternative way of getting sobi data

**Comment by fenuks** [comment]
Oh, I wish I knew they support gbfs! And thank you. There are few more systems that I wish to see supported, planning to add them in future.

**Pull request merged**